backend/flask_api.py

```diff
@@ -191,6 +192,62 @@ def delete_project(project_id):
         return make_response('Success')
 
 
+@app.route('/api/projects/<uuid:project_id>', methods=['PUT'])
+def update_project(project_id):
+    """Updates Project by ID
+
+    Args:
+        project_id, updated manifest in json/json5 format
+
+    Returns:
+        response: Success response
+                  or 404 if project is not found
+                  or 409 if project_id differs from manifestID
+                  or 500 in case of validation error
+    """
+    try:
+        res = coll.find_one({'_id': project_id})
+        if res is None:
+            raise ApiException("Project not found", 404)
+        elif request.is_json or request.headers['content-type'] == "application/json5":
```

Comment:
please check 'application/json5' in request.content_type instead, as it might be "application/json5;charset: utf-8"